src/utils/i18n.ts

```diff
@@ -3,11 +3,13 @@ import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 
 import en from "@/locales/en.json";
+import ko from "@/locales/ko.json";
```

Comment:
This is no longer needed - you can now register the language in `./src/utils/constants` and it will be available in the language switcher (and lazy-loaded).

Reply:
<img width="1470" alt="Screenshot 2025-02-21 at 18 00 04" src="https://github.com/user-attachments/assets/2149ffc1-8273-4f8a-95ea-e0d71daaf0b3" /> tested ko data locally.

Reply:
I'll check it. thank you!